### PR TITLE
feat(i18n): add Czech language

### PR DESCRIPTION
**Pull request by VentyCZ**
**Type:** translation

**Description:**

Add Czech translation - as per Readme ;)

### CODE REVIEW

**Comment by juanpicado** [comment]
Nice 👍

File: i18n/translations/pt-BR.json [hunk 1]
```diff
@@ -141,6 +141,7 @@
     },
     "lng": {
         "english": "Inglês",
+        "czech": "Tcheco",
```
Comment (priscilawebdev) [inline i18n/translations/pt-BR.json:144]: <3

**Comment by priscilawebdev** [comment]
@juanpicado  I try hahaha :)

**Comment by VentyCZ** [comment]
> @juanpicado I try hahaha :)

@priscilawebdev Thanks for merging both of my pull-requests!
I don't want to bloat a closed issue, but I just wanted to ask: How come you are learning czech ? You know, as opposed to other languages ? Just curious :)

**Pull request merged**